web/styles/zulip.css

```diff
@@ -2071,3 +2071,12 @@ body:not(.hide-left-sidebar) {
         right: -2px;
     }
 }
+
+/* Only show links on the trailing bookend. */
+.channel-settings-link {
+    display: none;
+}
+
+.trailing_bookend .channel-settings-link {
+    display: inline;
+}
```

Comment:
`channel-settings-link` is pretty generic as a name; `bookend-channel-settings-link` seems like a better name.